README.md

```diff
@@ -6,6 +6,11 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
 
 [elixir-companies]: https://github.com/doomspork/elixir-companies
 
+#### Accessories
```

Comment:
Should this just go in the E-Commerce section? 💭 
https://github.com/jah2488/elm-companies#e-commerce
Yes, they make/sell Accessories, but the ultimate function of their website is selling, i.e. E-Commerce.

Great addition BTW! 👍

Reply:
I moved it, thanks.